tests/suites/relations/relation_data_exchange.sh

```diff
@@ -6,22 +7,25 @@ run_relation_data_exchange() {
 
 	ensure "${model_name}" "${file}"
 
-	# Deploy 2 wordpress instances and one mysql instance
-	juju deploy wordpress -n 2 --force --series jammy
-	wait_for "wordpress" "$(idle_condition "wordpress" 0 0)"
-	wait_for "wordpress" "$(idle_condition "wordpress" 0 1)"
+	echo "Deploy 2 wordpress instances and one mysql instance"
+	juju deploy wordpress -n 2 --force --series bionic
 	# mysql charm does not have stable channel, so we use edge channel
-	juju deploy mysql --channel=edge --force --series jammy
+	juju deploy mysql --channel=edge --force --series focal
+	wait_for "wordpress" "$(idle_condition "wordpress" 1 0)"
+	wait_for "wordpress" "$(idle_condition "wordpress" 1 1)"
 	wait_for "mysql" "$(idle_condition "mysql")"
 
-	# Establish relation
+	echo "Establish relation"
 	juju relate wordpress mysql
```

Comment:
I would include the relate command before the `wait_for` statements as well, since this may take an arbitrary amount of time and interfere with the later exec